Arrowgene.Ddon.Shared/Entity/PacketStructure/C2SInstance_13_46_16_Ntc.cs

```diff
@@ -0,0 +1,43 @@
+using Arrowgene.Buffers;
+using Arrowgene.Ddon.Shared.Entity.Structure;
+using Arrowgene.Ddon.Shared.Model;
+using Arrowgene.Ddon.Shared.Network;
+
+namespace Arrowgene.Ddon.Shared.Entity.PacketStructure
+{
+    public class C2SInstance_13_46_16_Ntc : IPacketStructure
+    {
+        public PacketId Id => PacketId.C2S_INSTANCE_13_46_16_NTC;
```

Comment:
Do you know when these get sent? Is it from touching stuff? Must be either some OM interaction or something we triggered where the client wants us to provide some behavior.